src/components/notes/EditorCorner.jsx

```diff
@@ -7,13 +7,22 @@ import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 
 import SharingWidget from 'components/notes/sharing'
 
+// https://mui.com/components/tooltips/#custom-child-element
+const ForwardedIcon = React.forwardRef(function ForwardedIcon(props, ref) {
+  return (
+    <div ref={ref}>
+      <Icon {...props} />
+    </div>
+  )
+})
+
 const EditorCorner = ({ doc, isPublic, isReadOnly, title }) => {
   const { t } = useI18n()
   if (!isPublic) return <SharingWidget file={doc.file} title={title} />
   else if (isReadOnly) {
     return (
       <Tooltip title={t('Notes.Editor.read_only')}>
-        <Icon icon="lock" color="var(--primaryTextColor)" />
+        <ForwardedIcon icon="lock" color="var(--primaryTextColor)" />
```

Comment:
Why not updating cozy-ui in order to forwardRef by default on Icon? 🤔

Reply:
I don't remember why I didn't choose to update cozy-ui, maybe there were more problems. I can do that though

Reply:
Did you created the issue on cozy-ui? Can you link it in the code?

Reply:
I didn't create an issue yet, the build is failing with the latest cozy-ui for a different reason (konnector related)

Reply:
I mean, I'm not sure the forwardRef error exists with the latest cozy-ui (that doesn't work on cozy-notes right now, I prefered delivering the fixes instead)

Reply:
I will check on the latest version after cozy-notes is delivered

Reply:
https://github.com/cozy/cozy-ui/issues/1998